programming/Encoders.md

```diff
@@ -0,0 +1,69 @@
+# Encoders
+
+
+**Encoders measure how fast a motor spins.**<br> Which is immeasurably important, it's the reason why our robot can actually turn and still drive straight.  This is because of the fact we can precisely measure how quickly our motor is turning and what angle it's turned in which allows us to make sure we have all our wheels facing the right direction.  Without encoders our robot would easily just end up flipping over due to the fact half of the motors were facing the wrong direction. 
```

Comment:
Instead of using a `<br>`, can you separate these into their own paragraphs?

Reply:
yep.